one-time-project-setup.md

```diff
@@ -43,15 +43,15 @@ Create a file named `.env`.
 The front-end layer needs to send API requests to the back-end layer. In order to handle this, the front-end layer repo **must** include a `.env` file with this line:
 
 ```
-REACT_APP_BACKEND_URL=http://localhost:5000
+VITE_APP_BACKEND_URL=http://localhost:5000
 ```
 
-Note that this `REACT_APP_BACKEND_URL` _must_ include `http://`.
+Note that this `VITE_APP_BACKEND_URL` _must_ include `http://`.
 
-Use this environment variable to send your API requests. You can read it by using the expression `process.env.REACT_APP_BACKEND_URL`. For example, we may use it like this in any component:
+Use this environment variable to send your API requests. You can read it by using the expression `import.meta.env.VITE_APP_BACKEND_URL`. For example, we may use it like this in any component:
 
 ```js
-axios.get(`${process.env.REACT_APP_BACKEND_URL}/boards`, {
+axios.get(`${import.meta.env.VITE_APP_BACKEND_URL}/boards`, {
```

Comment:
Is this typically how the value would be imported and used, or would it be imported at the top of the file and referenced by `VITE_APP_BACKEND_URL`? 

My concern is if we show this example and it's one of their first intros to reading from a .env in javascript, we're going to see students writing their code like this.

Reply:
Are you saying to do something like this: 
```js
// Top of file
import { useState } from 'react'
import reactLogo from './assets/react.svg'
import viteLogo from '/vite.svg'
import './App.css'

function App() {
    const VITE_APP_BACKEND_URL = import.meta.env.VITE_APP_BACKEND_URL
    axios.get(`${VITE_APP_BACKEND_URL}/boards`, { ...
```

I'm not too familiar with the `import.meta.env` syntax but what I saw in the [Vite documentation](https://vite.dev/guide/env-and-mode#env-variables) and online I saw people using it like that. I didn't see anything that resembled more of what we have with other imports above.

Reply:
Thanks for digging in and sharing the docs link! I was wondering if the variables should be imported at the top of the file, but I looked up best practices articles from the past year and they show similar syntax, sometimes pulling it out into it's own variable as you're showing in the example above. 

I think I prefer it with the variable on it's own line so there is less happening in the `axios` call line, but that is personal preference, the example is great as-is.

Reply:
I changed to the example above! That way it encourages DRY code since they will most likely be using imports like that multiple times in the same file.